code/game/objects/items.dm

```diff
@@ -1092,4 +1092,69 @@ var/global/list/image/blood_overlays = list()
 	if(shrapnel_type)
 		return new shrapnel_type(src)
 	else
-		return 0
\ No newline at end of file
+		return 0
+
+/obj/item/duplicate(atom/destination)
+	var/obj/item/dupe = ..()
+
+	if(iscarbon(loc) && iscarbon(destination))
```

Comment:
Why is this required? Shouldn't human and monkey code just be duplicating clothes items itself? This is a serious break of encapsulation.